spec/workers/additional_email_confirmation_worker_spec.rb

```diff
@@ -1,6 +1,6 @@
-require "spec_helper"
+require "rails_helper"
 
-describe AdditionalEmailConfirmationWorker do
+RSpec.describe AdditionalEmailConfirmationWorker, type: :job do
```

Comment:
Just [looked up job specs](https://relishapp.com/rspec/rspec-rails/docs/job-specs/job-spec), because I was concerned it wasn't applicable since we aren't using ActiveJob.

It seems like the `type: job` _is_ targeted at ActiveJob, and adds additional functionality for it - but since this isn't causing any problems, I don't see a reason not to use it.

Ignore this comment 🙂